src/views/RoomDetail.vue

```diff
@@ -5,21 +5,28 @@
         <h2 class="my0">{{ this.room.roomName }}</h2>
       </b-col>
       <b-col cols="4" class="h-100 d-flex justify-content-center">
-        <radial-progress-bar :diameter="55" :completed-steps="completedSteps" :total-steps="totalSteps()"
+        <radial-progress-bar v-if="this.room.checkIns.user" :diameter="55" :completed-steps="this.room.checkIns.user.length" :total-steps="totalSteps()"
```

Comment:
Die Progressbar sollte auch angezeigt werden, wenn 0 Personen im Raum sind. Dann sollte halt 0 / capacity da stehen.

Reply:
Kann man machen, finde das aber auch eine Option das gar nicht anzuzeigen. Aber wenn du das so willst.

Reply:
Bei derAnzeige für den morgigen Tag finde ich das ok, wenn die Leiste nicht angezeigt wird, aber beim heutigen ist es glaub gut zu wissen, das halt 0/kapazität drin sind.

Reply:
In Teams geklärt. 👍